cif_rho.dic

```diff
@@ -2124,6 +2038,111 @@ save_atom_rho_multipole_radial_slater.n3_su
 
 save_
 
+save_atom_rho_multipole_radial_slater.n_list
+
+    _definition.id                '_atom_rho_multipole_radial_slater.n_list'
+    _alias.definition_id          '_atom_rho_multipole_radial_slater_n_list'
+    _definition.update            2024-04-23
+    _description.text
+;
+    These items are used when the radial dependence of the valence
+    electron  density, R(kappa'(l),l,r), of the atom specified in
+    atom_rho_multipole.atom_label is expressed as a Slater-type
+    function [Hansen & Coppens (1978), equation (3)]:
+
+    R(kappa'(l),l,r) = [{zeta(l)\}^\{n(l)+3\}^/\{n(l)+2\}!]\
+                        *(kappa'(l)*r)^n(l)^
+                        *exp(-kappa'(l)*zeta(l)*r)
```

Comment:
The curly braces here should probably not be escaped with backslahes,

Reply:
Agree.